src/library_webgl2.js

```diff
@@ -298,6 +305,8 @@ var LibraryWebGL2 = {
     }
     {{{ makeSetValue('params', '0', 'ret', 'i32') }}};
   },
+  glGetQueryObjectivEXT: 'glGetQueryObjectuiv',
+  glGetQueryObjectuivEXT: 'glGetQueryObjectuiv',
```

Comment:
Each aliased function should have a corresponding `funcname__sig: 'signaturehere'` field that specifies the function signatures of the aliases. That will help the aliasing backend use a better implemented aliasing function.

Reply:
Done (the aliases are no longer there, I chose a different approach).